docs/modules/data-structures/pages/cpmap.adoc

```diff
@@ -0,0 +1,47 @@
+= CPMap
+[[cpmap]]
+
+Hazelcast `CPMap` is a distributed implementation of a minimal key-value interface. It supports 
+atomic `compareAndSet`, `delete`, `get`, `put`, `remove` and `set` operations. Since `CPMap` is a 
+distributed key-value implementation, these operations involve remote calls and thus their 
+performance differs from non-distributed key-value data structures, e.g. `java.util.Hashmap`. 
+`CPMap` is an enterprise only feature.
+
+WARNING: There is no unsafe variant of `CPMap`, unlike other CP data structures. Therefore, CP must
+be enabled before making use of `CPMap`.
+
+The following example creates an instance of a `CPMap` in the `default` CP group, then updates its
+contents via the operations listed previously.
+
+[source,java]
+----
+include::ROOT:example$/dds/cpmap/ExampleCPMap.java[tag=cpm]
+----
+
+The mutations over the `CPMap` `capitalCities` in the above example are replicated across each of 
+the members of the `default` CP Group. 
+
+Preference should be given to `set` for associating a key with a value rather than `put` when the 
```

Comment:
avoid "should"

Use `set` to associate a key with a value instead of `put` when the previously-associated....

Reply:
Done.